src/go-postgres/README.md

```diff
@@ -7,7 +7,7 @@ Use and develop Go + Postgres applications. Includes appropriate runtime args, G
 
 | Options Id | Description | Type | Default Value |
 |-----|-----|-----|-----|
-| imageVariant | Go version: | string | 1.22-bookworm |
+| imageVariant | Go version: | string | 1.23-bookworm |
```

Comment:
Once this PR is merged, https://github.com/devcontainers/action will run automatically ; which opens a PR that updates the README file based on `devcontainer-feature.json` and `NOTES.md`.

Hence, can we revert the changes to this file to avoid merge conflicts? Thanks!

Reply:
@gauravsaini04 Can you help resolve this?